eval/http.go

```diff
@@ -84,7 +85,7 @@ func SetGetBody(req *http.Request, bodyLimit int64) error {
 	return nil
 }
 
```

Comment:
Return a warning diagnostic and log on callee side? So you can apply "Block" context.